modules/react/tabs/lib/TabsList.tsx

```diff
@@ -17,7 +17,8 @@ import {
 import {useTabsModel} from './useTabsModel';
 
 // Use `Partial` here to make `spacing` optional
-export interface TabListProps<T = unknown> extends Partial<ExtractProps<typeof Stack, never>> {
+export interface TabListProps<T = any>
+  extends Omit<Partial<ExtractProps<typeof Stack, never>>, 'children'> {
```

Comment:
I wonder if `ExtractProps` should remove children... It is easier to add `children` than to take it away. I don't like using `Omit` as the type becomes much messier. All the `Extract*` utility types avoid using it and instead return clean interfaces